src/Pages/Help/Keyboard.elm

```diff
@@ -58,16 +62,93 @@ subscriptions model =
 
 
 
+-- Markdown
+
+
+content : Html msg
+content =
+    Markdown.toHtml [ class "markdown" ] """
+
+# Key Binds                     
+
+You can use your keyboard to control Kodi, this is the default setting but you can change what the keyboard 
+
+controls on the [settings page]. The available options are:
+
+Kodi
+---------
+The keyboard controls Kodi and default keyboard interaction with the browser is disabled. (eg. using up and down 
+
+arrows to scroll a page). [Browser command] = [Kodi action].
+
+~~~
+Cursor LEFT = Direction LEFT
+
+Cursor RIGHT = Direction RIGHT
+
+Cursor UP = Direction UP
+
+Cursor DOWN = Direction DOWN
+
+BACKSPACE = Back
+
+ENTER = Select
+
+TAB = Close
+
+SPACE BAR = Play/Pause
+
+Key "C" = Context menu
+
+Key "+" = Volume Up
+
+Key "-" = Volume Down
+
+Key "X" = Stop
+
+Key "T" = Toggle subtitles
+
+Key ">" = Play Next
+
+Key "<" = Play Prev
+
+Key "" = Full screen
+
+~~~
+[Got improvements to add? click here]
+
+Browser
+---------
+The keyboard controls the browser only. When the [remote] is open the keys will control only Kodi.
+
+Both
+---------
+Keyboard commands are executed in Kodi and in the browser. When the [remote] is open the keys will control only 
+
+Kodi. NOTE: Many commands are shared by both the browser and Kodi. Eg. Backspace.
+
+** **       
+
+[remote]: http://localhost:1234/#remote
+[settings page]: http://localhost:1234/settings/web
+[Got improvements to add? click here]:https://github.com/xbmc/chorus2/blob/master/src/js/apps/input/input_app.js.coffee
```

Comment:
This will need to be a different url

Reply:
Can you please specify which link are you talking about?

Reply:
All links that lead to the old chorus repo